.github/workflows/publish_pypi.yaml

```diff
@@ -0,0 +1,25 @@
+name: Publish to PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
```

Comment:
@hechth Is there a specific reason to use `v1`? I'm curious because [`v2`](https://github.com/actions/setup-python/releases/tag/v2.0.0) and [`v3`](https://github.com/actions/setup-python/releases/tag/v3.0.0) are already available.

Reply:
No there is not.